### PR TITLE
Completed html-forms exercise. -Paul Glorioso

### PR DESCRIPTION
**Pull request by PaulGlorioso**
Forms

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <div>
+        <label for="email"
+          >Email: <br />
```
Comment (BrettAlbright) [inline html-forms/index.html]: I would replace the br tags with a div wrapped around the input. br tags aren't really used anymore except sometimes for text formatting.

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+          <input type="text" id="email" />
+        </label>
+      </div>
+      <div>
+        <label for="why"
+          >Why is coding cool?<br />
+          <select>
```
Comment (BrettAlbright) [inline html-forms/index.html]: you need to make sure to have all of the attributes such as name and an id which matches the `for` attribute of the label

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+            <option>Select an option</option>
+            <option value="interesting">Because it's interesting!</option>
+            <option value="fun">Because it's fun</option>
+            <option value="powerful">Because it's powerful</option>
+          </select>
+        </label>
+      </div>
+      <div>
+        <div>Favorite Language (Choose one)</div>
+        <input type="radio" id="html" name="language" value="html" /><label
```
Comment (BrettAlbright) [inline html-forms/index.html]: here you want to either put a for attribute on the labels which match the id attribute, or wrap each of the radio inputs with a label. That is to make sure the click functionality of the label works

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Very nice initial submission! There are a few updates to make, and make sure that each input has an appropriate `name` attribute on it.

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**